File: finddata/bubbleSort.py
from tkinter import Tk,Label
def bubbleSort(arr):
    items = arr[:]
    for i in range(len(items)):
        for j in range(i+1,len(items)):
            if items[j] < items[i]:
                items[j],items[i] = items[i],items[j]
    return items
        
    
arr = [6,8,9,3,1,2,5,7,0]
# messagebox.showinfo("Result",bubbleSort(arr))
root = Tk()
label = Label(root,text=bubbleSort(arr),fg='red',bg='white')
label.pack()
root.mainloop()

File: finddata/bubbleSort.js
function swap(value, index1, index2) {
    let temp = value[index1];
    value[index1] = value[index2]
    value[index2] = temp
    return temp;
}
function bubbleSort(arr) {
    for (let i = 0; i < arr.length; i++){
        for (let j = 0; j <= i; j++){
            if (arr[j] > arr[i]) {
                swap(arr,i,j)
            }
        }
    }
    return arr;
}
let arr = [9, 1, 4, 8, 3, 1, 9, 4]
console.log(bubbleSort(arr))